.github/workflows/dependabot.js.yml

```diff
@@ -0,0 +1,41 @@
+name: Dependabot
+
+on:
+  push:
+    branches: [ dependabot/npm_and_yarn/** ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12'
+    - name: Autofix lockfile
+      run: |
+        # restore yarn.lock from the previous commit
+        git checkout HEAD^ -- yarn.lock
+
+        # install yarn-plugin-deduplicate
+        yarn plugin import https://raw.githubusercontent.com/eps1lon/yarn-plugin-deduplicate/latest/bin/%40yarnpkg/plugin-deduplicate.js
+
+        # if package.json was not updated, upgrade the dependency
+        git diff --name-only HEAD^ HEAD | grep -q 'package.json' || yarn up `git log -1 --pretty=%s | awk '{ print $3 }'`
```

Comment:
It seems like there's a disconnect between the comment and the command.  It looks like we're looking if there were changes made to `package.json` and upgrading those dependencies.  Am I misunderstanding the command?

Reply:
@kevinbarabash If `grep` has succeeded, the `package.json` was updated, i.e., the version range is already updated, so it will install upgraded dependency on `yarn install` step and there is no need to run `yarn up`.